hnn_core/network.py

```diff
@@ -976,8 +983,6 @@ def _attach_drive(self, name, drive, weights_ampa, weights_nmda, location,
                         self.connectivity[-1]['src_gids'] = \
                             self.connectivity[-2]['src_gids']
 
-            seed_increment += 1
```

Comment:
was this a bug?

Reply:
Yes, though I don't think it actually affected anything since `seed_increment` is defined at the beginning of each iteration as the the enumerator in the for-loop.